aiida/transports/plugins/ssh.py

```diff
@@ -68,92 +68,97 @@ class SshTransport(Transport):  # pylint: disable=too-many-public-methods
     # I disable 'password' and 'pkey' to avoid these data to get logged in the
     # aiida log file.
     _valid_connect_options = [
-        ('username', {
-            'prompt': 'User name',
-            'help': 'user name for the computer',
-            'non_interactive_default': True
-        }),
+        (
+            'username', {
+                'prompt': 'User name',
+                'help': 'Login user name on the remote machine.',
+                'non_interactive_default': True
+            }
+        ),
         ('port', {
             'option': options.PORT,
-            'prompt': 'port Nr',
+            'prompt': 'Port number',
             'non_interactive_default': True
         }),
         (
             'look_for_keys', {
                 'switch': True,
                 'prompt': 'Look for keys',
-                'help': 'switch automatic key file discovery on / off',
+                'help': 'Automatically look for private keys in ~/.ssh (default: true)',
```

Comment:
No, the problem is exactly that it is not shown, e.g. in `verdi computer configure ssh -h`

Reply:
I think we should fix this by setting `show_default=True` in the option. Not sure if this intermediate layer of defining options through this dictionary already supports it, but piping that through should be easy enough

Reply:
just checked in the code, and it is almost possible, but requires a small fix in the code. I am making a PR now.

Reply:
Great! In that case we should obviously remove the defaults from the help string again.

Reply:
PR #4223 should fix the problem. This of course won't really work during the tutorial since the code won't be on the virtual machines, but then again, neither would be the changes to the help strings. So I guess it doesn't change anything and we can revert the hardcoded changes here

Reply:
Yes, let's remove all mentions of default values from here.

And while people won't get the updated AiiDA version during the tutorial, at least we'll be able to point them to the automatically generated verdi docs:
https://aiida.readthedocs.io/projects/aiida-core/en/latest/reference/command_line.html